modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -12929,6 +12929,7 @@ oauth2-allow-refresh-token-grant=Allow Refresh Token
 oauth2-allow-resource-owner-password-credentials-grant=Allow Resource Owner Password Credentials
 oauth2-application-external-reference-code=OAuth 2 Application External Reference Code
 oauth2-application-name=OAuth 2 Application Name
+oauth2-application-x-required-by-x=The OAuth2Application {0} is required by {1}
```

Comment:
Take a look at this https://github.com/brianchandotcom/liferay-portal/pull/58629#issuecomment-390004381 and use a Title or a sentence.